jbt-it/neomdb
# #29 create accounts for new user

Added nextcloud, mediawiki and mail account creation (currently commented out -> This is going to be resolved before the first deployment).
Added check if name of the user already exists in the database (then a user with v.nachname1 is going to be created).

How to test:
1. Use following POST method (after logging in):
`https://localhost:3030/users` with body
"{
    "vorname": "lennart",
    "nachname":"rukower",
    "name":"l.rukower",
    "geschlecht":"1",
    "password":"kgnjh7i5h4teg",
    "generation":"14",
    "traineeSeit":"2012-12-01",
    "email":"le.rukower@gmail.com",
    "geburtsdatum":"1990-06-06",
    "handy":"49079548954"
}" with an API client like thunder client for example
2. Call this route mutliple times and check in the databse if multiple instances of l.rukower were created (l.rukower, l.rukower1, l.rukower2 etc.)

Closes #29
